Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Events/ReceiverBase.cs

```diff
@@ -16,8 +17,23 @@ public abstract class ReceiverBase
 
         public bool HideUnityEvents;
         protected UnityEvent uEvent;
+
+        /// <summary>
+        /// Each Receiver has a base Event it raises, (in addition to others).
+        /// </summary>
+        public UnityEvent Event
```

Comment:
Why is this Event public set/get but the other specific inherited event types are only public getters?

Reply:
Good catch. Removed the setter.

Reply:
It's because in AddReceiver I am specifying generic type, and generic types cannot AFAIK be instantiated with params. So, I need to do:

```
            var interactableEvent = new InteractableEvent();
            var result = new T();
            result.Event = interactableEvent.Event;
```